src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/MessageDrivenSubscriptionsConfigExtensions.cs

```diff
@@ -33,12 +32,6 @@ public static void SubscriptionAuthorizer(this TransportExtensions transportExte
         {
             Guard.AgainstNull(nameof(authorizer), authorizer);
             var settings = transportExtensions.Settings;
-            var transport = settings.Get<TransportDefinition>();
-            if (transport.GetOutboundRoutingPolicy(settings).Publishes == OutboundRoutingType.Multicast)
-            {
-                var message = $"The transport {transport.GetType().Name} supports native publish-subscribe so subscriptions are not managed by the transport in the publishing endpoint. Use the native transport tools managing subscritions.";
-                throw new ArgumentException(message, nameof(authorizer));
-            }
 
             settings.Set("SubscriptionAuthorizer", authorizer);
```

Comment:
With the native pub-sub check removed here, is there a problem introduced by allowing the "SubscriptionAuthorizer" setting to be set when it it previously couldn't be?